atc/metric/emit.go

```diff
@@ -66,7 +66,13 @@ var (
 	emissions       chan eventEmission
 )
 
-func Initialize(logger lager.Logger, host string, attributes map[string]string) error {
+func Initialize(logger lager.Logger, host string, attributes map[string]string, bufferSize int) error {
```

Comment:
As we always expect `bufferSize` to be `uint` anyway, wdyt of changing the signature to take an `uint` instead of `int`?

```suggestion
func Initialize(logger lager.Logger, host string, attributes map[string]string, bufferSize uint32) error {
``` 

It seems to make that it'd be better to do the casting at the `make(chan ...` that comes later rather than at the moment that we pass the value to `Initialize` 🤔

Reply:
I will make this change - makes sense to me to constrain it with a uint32 and do the cast at the last possible moment.